.github/workflows/ci.yml

```diff
@@ -104,12 +109,12 @@ jobs:
     needs: test-modules
     strategy:
       matrix:
-        go-version: [1.20.x, 1.x]
         platform: [ubuntu-latest, macos-latest]
         module: [bigtable, cockroachdb, consul, datastore, firestore, nats, nginx, pubsub, spanner, toxiproxy]
     uses: ./.github/workflows/ci-test-go.yml
     with:
-      go-version: ${{ matrix.go-version }}
+      go-version: "1.x"
```

Comment:
Shall the condition to run golangci-lint be changed to be applied on 1.x or use 1.20.x instead here ?

Reply:
Will use 1.20.x here, thanks!